task-2/public/index.html

```diff
@@ -0,0 +1,192 @@
+            </p>
+            <p class="article__text">
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
+do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+enim ad minim veniam, quis nostrud exercitation ullamco laboris
+            </p>
+          </article>
+        </div>
+      </section>
+    </div>
+    <section class="sign-up-box">
+      <p class="sign-up-box__title">
+        Sign up for our newsletter!
+      </p>
+      <form class="input-form" method="POST" action="#" name="sign-up-form">
+        <input class="input-form__submit-icon" type="image" name="submit-button" id="subscribe-button" src="./assets/send-icon.svg">
+        <input class="input-form__email-input" type="text" name="user-email" id="email-input" required="required"/>
+        <label class="input-form__label" for="email-input">Enter a valid email address</label>
+      </form>
+    </section>
+    <div class="wrapper_page">
+      <section class="wrapper_content">
+        <div class="wrapper_line">
+          <article class="article">
+            <img class="article__promo" src="./assets/images/img-countryside.png" alt="">
+            <a class="article__category button_link" href="#">
+              lifestyle
+            </a>
+            <p class="article__title">
+              Top 10 song for running
+            </p>
+            <p class="article__text">
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
+do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+enim ad minim veniam, quis nostrud exercitation ullamco laboris
+            </p>
+          </article>
+          <article class="article">
+            <img class="article__promo" src="./assets/images/img-road.png" alt="">
+            <a class="article__category button_link" href="#">
+              lifestyle
+            </a>
+            <p class="article__title">
+              Cold winter days
+            </p>
+            <p class="article__text">
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
+do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+enim ad minim veniam, quis nostrud exercitation ullamco laboris
+            </p>
+          </article>
+        </div>
+        <button class="button_load">
+          <span class="button button__text">
+          Load more
+          </span>
+        </button>
+      </section>
+    </div>
+    <footer class="footer">
+      <div class="footer__wrapper">
+        <nav class="footer__links">
+          <a class="button_link footer__links_link-style" href="#">
+            Terms and conditions
+          </a>
+          <a class="button_link footer__links_link-style" href="#">
+            Privacy
+          </a>
+        </nav>
+        <div class="social">
+          <a class="social__links_link-style">
+            Follow
+          </a>
+          <a class="button_link social__links_icon social__links_icon-facebook " href="#">
```

Comment:
Should it really look like a link? Don't think that this is a good place for the `button-link` class.